### DISCUSSION TITLE
faux Psycho-pass system

### DISCUSSION THREAD

**Discussion by Pockets-byte** [unitystation/unitystation#7936] [Ideas]
A numeric system that gives a count on the probability that a player is either up to or will break the rules, or has would be very very useful, taking into account ingame actions and past history, actions like firing guns, carrying guns or weapons, access to dangerous areas of the station, antag status, past actions, killings, and so on, and putting that into a tentative number score for admins to see, would be a very helpful tool for gauging players at a glance without digging into logs, ideally it would ask take good behavior, into account, not making a mess of things,  performing their roles, repairing the station and so on, while it wouldn't be an authority on the intents of players and so on, it would be a good way to help single out players who have broken the rules previously, or need to be watched

**Comment by Alainx277** [comment]
Space station social credit 🤔

**Comment by jamessimo** [comment]
Whiles I don’t want to “score” people as that opens up a huge can of dystopian worms,
we could make “bans/kicks” available to server admins so they can better judge if a person who’s breaking rules has a past behavior of it.

**Comment by PerfectTangent** [comment]
I think Jamessimo's solution is the better, much more effective one.

**Comment by Pockets-byte** [comment]
moderation history... should be a given? but I don't necessarily see the issue with this being rather dystopic, considering it's already a complete dictatorship
maybe this sugrestion is a bit overblown, but a well improved alerting system and logging system is long overdue

**Comment by PerfectTangent** [comment]
> it would be a good way to help single out players who have broken the rules previously,

given this point made, it'd be redundant with moderation history and a replay system, making it unnecessary, and unlike those methods, this would be prone to false flags.

> but a well improved alerting system and logging system is long overdue

Make issues covering those systems.

Regardless, converting this to a discussion.
